notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>We use the <a href="https://cocodataset.org/#home" target="_blank">COCO</a> validation set (5,000 images) for evaluation. -&gt; The <a href="https://cocodataset.org/#home" target="_blank">COCO</a> validation set (5,000 images) is used for evaluation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>